### ISSUE TITLE
Serve playground from PR branches

### ISSUE THREAD

**Issue by nguerrera** [microsoft/cadl#451]
It would be awesome if we could share a link to the playground with our changes when making language server changes. This would let reviewers play around with the new behavior to see the effects of the proposed changes.

Apparently, @timotheeguerin and @bterlson discussed this already so I'll let them comment on possible approaches.

**Comment by timotheeguerin** [comment]
Yeah, easiest approach was just to publish the playground dist directory under the commit hash in the storage account and you'd be able to access it via `/pr/<shortcommit>/`. The one issue with this was how to clear old data if that even mattered.

**Comment by timotheeguerin** [comment]
What is a bit more tricky is making a comment on the pull request with the link. ADO provide a GithubComment task which lets you create a comment(autorest uses that to give try it links for each PR) however every build it will make a new comment instead of updating the one which gets create a lot of spam. Better solution would be to have a small script dealing with the comment ourself but then we do have to handle creating a github account and manage credentials for that.
